### PR TITLE
Change Footer Links In index.html

### LINKED ISSUE #846: Add the link of the image sequencer in the footer.
Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you :gift_heart: 

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!

:thinking: What you will need to know.
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

:clipboard: Step by Step
  :raising_hand_woman: Claim this issue: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

  :memo: Update: All that needs to be done is replace this line
image-sequencer/examples/index.html

Line 211 in
``` 
 This is an open source toolkit which you can help add to and improve on <a href="https://github.com/">Github</a>
```
with this line
```
 This is an open source toolkit which you can help add to and improve on <a href="https://github.com/publiclab/image-sequencer/issues">Github</a>
```
Line 202 in
``` 
 Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues">PublicLab.org</a>
```
with this line
```
 Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues/new">PublicLab.org</a>
```
Line 205 in 
```
<a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues">Ask a question &raquo;</a>
```

with this line
```
<a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues/new">Ask a question &raquo;</a>
```
  :floppy_disk: Commit your changes

  :twisted_rightwards_arrows: Start a Pull Request. There are two ways how you can start a pull request:

If you are familiar with the terminal or would like to learn it, here is a great tutorial on how to send a pull request using the terminal.

You can also edit files directly in your browser and open a pull request from there.

  :checkered_flag: Done Ask in comments for a review :)
:thinking::question: Questions?
Leave a comment below!

Is someone else already working on this?
We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. If someone seem stuck, offer them some help! Otherwise, take a look at some other issues you can help with. Thanks!

Thanks!!!

### PR DESCRIPTION
**Pull request by maks-ostashkevich**
Fixes #846

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
You can descriptively title the PR next time as I just did.

**Comment by HarshKhandeparkar** [comment]
You can add a link to the issue by using the format `#issue-number` e.g. `#123`. If you put the keywords {fix, fixes, resolve, resolves} etc. Before the issue number/link it will close the issue as soon as the PR is merged. 😊

File: examples/index.html [hunk 1]
```diff
@@ -199,16 +199,16 @@ <h2 style="margin-top:20px">Save</h2>
...
         <p>
-          <a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues">Ask a question &raquo;</a>
+          <a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues/new">Ask a question &raquo;</a>
         </p>
       </div>
       <div class="col-md-6">
         <h2>Improve this tool</h2>
         <p>
-          This is an open source toolkit which you can help add to and improve on <a href="https://github.com/">Github</a>
+          This is an open source toolkit which you can help add to and improve on <a href="https://github.com/publiclab/image-sequencer/issues">Github</a>
```
Comment (HarshKhandeparkar) [inline examples/index.html]:
```suggestion
          This is an open source toolkit which you can help add to and improve on <a href="https://github.com/publiclab/image-sequencer/">Github</a>
```

**Comment by maks-ostashkevich** [comment]
@publiclab/is-reviewers, asking for the reviews.

File: examples/index.html [hunk 2]
```diff
@@ -199,16 +199,16 @@ <h2 style="margin-top:20px">Save</h2>
       <div class="col-md-6">
         <h2>Need Help?</h2>
         <p>
-          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues">PublicLab.org</a>
+          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues/new">PublicLab.org</a>
```
Comment (HarshKhandeparkar) [inline examples/index.html]:
```suggestion
          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues/new">Gitter</a>
```

File: examples/index.html [hunk 3]
```diff
@@ -199,16 +199,16 @@ <h2 style="margin-top:20px">Save</h2>
...
         </p>
         <p>
-          <a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues">Ask a question &raquo;</a>
+          <a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues/new">Ask a question &raquo;</a>
```
Comment (HarshKhandeparkar) [inline examples/index.html]:
```suggestion
          <a class="btn btn-primary" href="https://gitter.im/publiclab/publiclab">Ask a question on Gitter</a>
```

**Comment by Harshithpabbati** [comment]
And ask @publiclab/is-reviewers to review your patch.

File: examples/index.html (hunk 2 above)
Comment (Harshithpabbati) [reply examples/index.html]: Hey why did you make it gitter??

Comment (HarshKhandeparkar) [reply examples/index.html]: All the links being the same makes no sense

**Comment by maks-ostashkevich** [comment]
@Harshithpabbati, do I have to ask someone particular?
@publiclab/is-reviewers gives no link in my comments.

**Comment by HarshKhandeparkar** [comment]
@jywarren?

**Comment by maks-ostashkevich** [comment]
@publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
Mentioning @publiclab/is-reviewers somwtimes doesn't work. You can mention someone specifically if you want and they might be able to mention `is-reviewers` themselves.

**Comment by maks-ostashkevich** [comment]
@jywarren

**Comment by Harshithpabbati** [comment]
That might be the way

**Comment by HarshKhandeparkar** [comment]
Harshith, is this ready to merge? Do you approve this?

**Comment by maks-ostashkevich** [comment]
@HarshKhandeparkar, it shows red "X" on the second suggestion, it this a trouble?

**Comment by HarshKhandeparkar** [comment]
No. You always have to look at the latest commit. If it has a green tick then it is good to go. Btw the middle one was cancelled as a new one was commited soon after.

**Comment by maks-ostashkevich** [comment]
@HarshKhandeparkar, last commit is "Changes approved"?
It still states that "Merging is blocked".

**Comment by HarshKhandeparkar** [comment]
Harshith, if you approve the pr you can add the `ready-to-merge` label. Thanks.

**Comment by Harshithpabbati** [comment]
@maks-ostashkevich thanks for your contributions we need more from you.

**Comment by jywarren** [comment]
Great work!!!

**Pull request merged**